examples/src/debugger/gizmos.rs

```diff
@@ -60,9 +60,9 @@ pub fn draw_triangles_debug_data_gizmos(
 
     // Draw last changes boundaries
     gizmos.rect(
-        debug_data.current_changes_bounds.0,
+        Vec3::from_array(debug_data.current_changes_bounds.0.to_array()),
         Quat::IDENTITY,
-        debug_data.current_changes_bounds.1,
+        Vec2::from_array(debug_data.current_changes_bounds.1.to_array()),
```

Comment:
Same, we don't need the conversion
```suggestion
        debug_data.current_changes_bounds.0,
        Quat::IDENTITY,
        debug_data.current_changes_bounds.1,
```